### PR TITLE
Added link to download PointNav Gibson v2

### PR DESCRIPTION
**Pull request by mathfac**
## Motivation and Context
Added link to download PointNav Gibson v2

## Types of changes

- Docs change / refactoring / dependency upgrade

### CODE REVIEW

File: configs/datasets/pointnav/gibson_v2.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ENVIRONMENT:
+  MAX_EPISODE_STEPS: 500
+SIMULATOR:
+  AGENT_0:
+    HEIGHT: 0.88
+    RADIUS: 0.18
+
+DATASET:
+  TYPE: PointNav-v1
```
Comment (Skylion007) [inline configs/datasets/pointnav/gibson_v2.yaml]: Shouldn't we reserve v2 for functional changes to the dataset class? Maybe it more like PointNav 1.1?

File: configs/datasets/pointnav/gibson_v2.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  SPLIT: train
+  DATA_PATH: data/datasets/pointnav/gibson/v1/{split}/{split}.json.gz
```
Comment (erikwijmans) [inline configs/datasets/pointnav/gibson_v2.yaml]:
```suggestion
  DATA_PATH: data/datasets/pointnav/gibson/v2/{split}/{split}.json.gz
```

File: README.md [hunk 3]
```diff
@@ -158,6 +158,7 @@ Download the Habitat related Gibson dataset following the instructions [here](ht
 | Task | Scenes | Link | Extract path | Config to use | Archive size |
 | --- | --- | --- | --- | --- | --- |
 | [Point goal navigation](https://arxiv.org/abs/1807.06757) | Gibson | [pointnav_gibson_v1.zip](https://dl.fbaipublicfiles.com/habitat/data/datasets/pointnav/gibson/v1/pointnav_gibson_v1.zip) | `data/datasets/pointnav/gibson/v1/` |  [`datasets/pointnav/gibson.yaml`](configs/datasets/pointnav/gibson.yaml) | 385 MB |
+| [Point goal navigation for sim2real](https://arxiv.org/abs/1912.06321) | Gibson | [pointnav_gibson_v2.zip](https://dl.fbaipublicfiles.com/habitat/data/datasets/pointnav/gibson/v2/pointnav_gibson_v2.zip) | `data/datasets/pointnav/gibson/v2/` |  [`datasets/pointnav/gibson_v2.yaml`](configs/datasets/pointnav/gibson.yaml) | 274 MB |
```
Comment (dhruvbatra) [inline README.md]:
```suggestion
| [Point goal navigation corresponding to Sim2LoCoBot experiment configuration](https://arxiv.org/abs/1912.06321) | Gibson | [pointnav_gibson_v2.zip](https://dl.fbaipublicfiles.com/habitat/data/datasets/pointnav/gibson/v2/pointnav_gibson_v2.zip) | `data/datasets/pointnav/gibson/v2/` |  [`datasets/pointnav/gibson_v2.yaml`](configs/datasets/pointnav/gibson.yaml) | 274 MB |
```

File: configs/datasets/pointnav/gibson_v2.yaml (hunk 1 above)
Comment (mathfac) [reply configs/datasets/pointnav/gibson_v2.yaml]: Great question, class itself wasn't changed. That's why v1 should be fine.

**Pull request merged**